lib/board.rb

```diff
@@ -0,0 +1,94 @@
+
+class Board
+
+    attr_accessor :dimensions
+
+   
+    def initialize()
+        @dimensions = []
+        for i in 0..2
```

Comment:
I am not sure how you are using `dimensions` but I think we are complicating the logic for our `initialize` method here.